packages/pose-core/src/factories/setter.ts

```diff
@@ -157,13 +160,15 @@ const createPoseSetter = <V, A, C, P>(
     resolveTarget,
     transformPose,
     posePriority,
-    convertTransitionDefinition
+    convertTransitionDefinition,
+    isReducedMotion
   } = setterProps;
 
   return (next: string, nextProps: Props = {}, propagate: boolean = true) => {
     const { children, values, props, activeActions, activePoses } = state;
     const { delay = 0 } = nextProps;
     const hasChildren = children.size;
+    const useTransition = !isReducedMotion() || isDev;
```

Comment:
after thinking about it a little I don't think that forcing animation in dev is a good thing, it's good to test those things out during development too and it would be weird to have pose animations on and everything else off

I realize that somebody might be surprised about their animations being off when forgetting that they use reduced motion setting - but it seems to me that one time console.warn in dev mode when `isReducedMotion() === true` would be sufficient to help people realize this.